.circleci/config.yml

```diff
@@ -267,6 +274,23 @@ jobs:
       - deploy-to:
           environment: production
 
+  veracode-scan:
+    executor: sast-orb/default
```

Comment:
have you tried using the test-executor here. not sure you need to use the orbs executor